sbws/util/config.py

```diff
@@ -70,7 +70,8 @@ def _get_user_config(args, conf=None):
             # sbws should start with a logger before reading configurations.
             print('Configuration file %s not found, using defaults.' %
                   args.config)
-        return _extend_config(conf, args.config)
```

Comment:
I think it would be more clear if you return conf in the if
```suggestion
            return conf
```